homu/main.py

```diff
@@ -180,14 +180,16 @@ def parse_commands(body, username, repo_cfg, state, my_username, db, *, realtime
             else:
                 cur_sha = sha
 
+            approver = word[len('r='):] if word.startswith('r=') else username
```

Comment:
`approver` seems to be a nice choice of the variable name! I should also consider renaming the entire `approved_by`/`Approved by` appearances to `approver`.